CMake/windows_config.cmake

```diff
@@ -40,6 +40,40 @@ macro(os_set_flags)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MP")
 
+        if (ENABLE_SECURITY_FLAGS)
```

Comment:
When this is false, what will be the value of SECURITY_COMPILER_FLAGS ?
Did we set it somewhere else?
we do use it even of it's false right?

Reply:
If we don't set it in our cmake, its value will be OFF by default (as defined in lrs_options), so we won't have the security flags (same thing if specifying False), if we set -DENABLE_SECURITY_FLAGS=True we will have the flags enabled

Reply:
I meant that if this flag is false than 
SECURITY_COMPILER_FLAGS
is not set
and used somewhere else, right?

Reply:
Oh, if this flag is false than SECURITY_COMPILER_FLAGS defaults to "" (empty string)
I can explicitly set it before the if, but i don't think it's a must